extension/vector/test/test_files/transaction.test

```diff
@@ -43,7 +43,7 @@
 -STATEMENT CALL CREATE_VECTOR_INDEX('embeddings', 'e_hnsw_index','vec', metric := 'l2');
 ---- error
 Binder exception: CREATE_VECTOR_INDEX is only supported in auto transaction mode.
-# -STATEMENT CALL QUERY_VECTOR_INDEX('embeddings', 'e_hnsw_index',CAST([0.1521,0.3021,0.5366,0.2774,0.5593,0.5589,0.1365,0.8557],'FLOAT[8]'), 3) RETURN nn.id ORDER BY distance;
+# -STATEMENT CALL QUERY_VECTOR_INDEX('embeddings', 'e_hnsw_index',CAST([0.1521,0.3021,0.5366,0.2774,0.5593,0.5589,0.1365,0.8557],'FLOAT[8]'), 3) RETURN node.id ORDER BY distance;
```

Comment:
btw these are commented out tests.

Reply:
Yeah I was keeping them for transaction tests. Should be enabled once we support creating index in manual transactions.